templates/partners/ihv-and-oem.html

```diff
@@ -6,149 +6,218 @@
         <li class="p-list__item is-ticked">Marketing funding for joint events</li>
         <li class="p-list__item is-ticked">Logos and joint branding</li>
       </ul>
     </div>
-    <div class="col-4 col-medium-3 p-card--highlighted">
-      <h3 class="p-card__title p-heading--4">Dedicated partner team</h3>
-      <ul class="p-list">
+  </div>
+  <div class="row">
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-4 col-start-medium-3">
+        <hr class="p-rule">
+      </div>
+    </div>
+    <div class="col-3 col-start-large-4 col-medium-1 col-start-medium-3">
+      <h3 class="p-heading--5">Dedicated partner team</h3>
+    </div>
+    <div class="col-6 col-medium-3">
+      <ul class="p-list--divided">
         <li class="p-list__item is-ticked">Named resources within Canonical for alliances, sales, technical training, marketing and certification</li>
       </ul>
     </div>
-    <div class="col-4 col-medium-3 p-card--highlighted">
-      <h3 class="p-card__title p-heading--4">Reference architecture</h3>
-      <ul class="p-list">
+  </div>
+  <div class="row">
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-4 col-start-medium-3">
+        <hr class="p-rule">
+      </div>
+    </div>
+    <div class="col-3 col-start-large-4 col-medium-1 col-start-medium-3">
+      <h3 class="p-heading--5">Reference architecture</h3>
+    </div>
+    <div class="col-6 col-medium-3">
+      <ul class="p-list--divided">
         <li class="p-list__item is-ticked">Develop joint hardware-based reference architectures for predictable and repeatable customer deployments</li>
       </ul>
     </div>
-    <div class="col-4 col-medium-3 p-card--highlighted">
-      <h3 class="p-card__title p-heading--4">Roadmap alignment</h3>
-      <ul class="p-list">
+  </div>
+  <div class="row">
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-4 col-start-medium-3">
+        <hr class="p-rule">
+      </div>
+    </div>
+    <div class="col-3 col-start-large-4 col-medium-1 col-start-medium-3">
+      <h3 class="p-heading--5">Roadmap alignment</h3>
+    </div>
+    <div class="col-6 col-medium-3">
+      <ul class="p-list--divided">
         <li class="p-list__item is-ticked">Weekly alignment calls with alliances and certification team</li>
         <li class="p-list__item is-ticked">Bi-annual product roadmap review sessions to plan for upcoming releases</li>
       </ul>
     </div>
   </div>
+
 </section>
 
-<section class="p-strip is-deep is-bordered">
+<section class="p-section">
   <div class="row">
-    <div class="col-6 u-vertically-center u-hide--small u-hide--medium">
-      <img src="https://assets.ubuntu.com/v1/729c18b3-certificates.svg" alt="Certified Ubuntu">
-    </div>
-    <div class="col-6">
+    <hr class="p-rule">
+  </row>
+  <div class="row--50-50">
+    <div class="col">
       <h2>Increase your sales by becoming Ubuntu Certified</h2>
```

Comment:
```suggestion
      <h2>Increase your sales by<br class="u-hide--small"> becoming Ubuntu Certified</h2>
```
Looks like there's supposed to be a break in the original design, but no idea if that's relevant anymore

Reply:
well spotted, thanks - it needs to be before the word "by"